crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -382,6 +386,9 @@ type MoveModuleEdge {
 
 type MoveModuleId {
 	name: String!
+	"""
+	Fetches the package that this Move module was defined in
```

Comment:
nit: I would remove the `Fetches` -- GraphQL is a declarative API -- this is just "the package that this Move module was defined in" -- we leave the fetching to GraphQL.